### ISSUE TITLE
Support for deploying S3 files from perform-tasks file

### ISSUE THREAD

**Issue by OlafConijn** [OlafConijn/AwsOrganizationFormation#52]
Would be nice to be able to upload files to S3 using a tasks file.
Specific scenario: uploading trusted IP list for GuardDuty, but i bet there is many others:

```
ServerlessWorkload:
  Type: s3-put-object                         # type
  LocalFile: ./myfile.ext                     # path to file, used to calculate hash
  RemotePath: s3://my-bucket/path/to/object
  OrganizationBinding:
    Account: !Ref AccountA
```

Questions:
- Does it make sense to support directories? is there a usecase?
- Would be nice to support pseudo parameters in the RemotePath, but is it really needed
- Any other suggestions w/ regards to configuration?

**Comment by tstroband** [comment]
How about this:
```
SomeStuffForS3:
   Type: copy-to-s3
   LocalPath: ./mydirorfile
   RemotePath: s3://my-bucket/path/to/object
   Overwrite: true
   ZipBeforePut: true
   OrganizationBinding:
      Account: !Ref AccountA
```

**Comment by OlafConijn** [comment]
implemented in version 0.9.4
